jet_tracking/jt_configs/mfx_rayonix_config.yml

```diff
@@ -2,17 +2,18 @@ api_msg:
   port: 5000
 
 hutch: 'mfx'
-experiment: 'mfxx45919'
-run: '1'
-sim: false
-ffb: true
+experiment: 'mfxlw7519'
+run: '17'
+sim: true
+ffb: false
```

Comment:
Running offline for this experiment?

But also should `sim` be renamed to `offline`? And is there a case when it would not be one or the other between these two options?

Reply:
Good  point, this will make it a little bit less confusing. I'll modify this.